### DISCUSSION TITLE
你午八还没上完吗？@VacuolePaoo

### DISCUSSION THREAD

**Discussion by Buer-Nahida** [share121/inter-knot#2022] [其他]
![x](https://private-user-images.githubusercontent.com/76843407/357630860-78cdae03-724c-44da-bb5b-fbd23607941a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM1OTY0NDQsIm5iZiI6MTcyMzU5NjE0NCwicGF0aCI6Ii83Njg0MzQwNy8zNTc2MzA4NjAtNzhjZGFlMDMtNzI0Yy00NGRhLWJiNWItZmJkMjM2MDc5NDFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE0VDAwNDIyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFiYTg3ZGE3OWQ2OWQ3ZGZhMDA1Y2M2YWViMTg5MDFmNTVjNzg3MjdhMjI2M2JmYTlkZmNhNDJlNjE2NjcyYTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1aR2RkbYDL24fGRnjkZmpxyZVAmT9IF-UFhDo-3tyJU)
# 你午八还没上完吗？@VacuolePaoo 
###### 你午八还没上完吗？@VacuolePaoo 
- 你午八还没上完吗？@VacuolePaoo 
- [x] 你午八还没上完吗？@VacuolePaoo 
1. 你午八还没上完吗？@VacuolePaoo 
> 你午八还没上完吗？@VacuolePaoo 
```rust
fn 你午八还没上完吗？() {}
```
@VacuolePaoo 
`你午八还没上完吗？`
@VacuolePaoo 
<kbd>你午八还没上完吗？</kbd>
@VacuolePaoo

**Comment by AetherLude** [comment]
# 玩原神去了

**Comment by Myitian** [comment]
u1s1，我大学早八上到9:40（就是两节45分连一起的）

**Reply by inkluv** [reply]
符玄大人原来在上大学吗。

**Comment by Myitian** [comment]
# 《午八》
![图片](https://github.com/user-attachments/assets/a8d56077-01d4-4514-8aa5-9afbdb05b0af)

**Comment by VacuolePaoo** [comment]
上万午八了，上厕硕

**Reply by cpu644** [reply]
快进到晚八

**Comment by nostalgiatan** [comment]
最近好像发生了好多事情，有点没绷住。
